src/views/account/Lessons/Lessons.tsx

```diff
@@ -43,6 +46,9 @@ const Lessons: Screen<"Lessons"> = ({ route, navigation }) => {
   const loadedWeeks = useRef<Set<number>>(new Set());
   const currentlyLoadingWeeks = useRef<Set<number>>(new Set());
 
+  const [shouldShowWeekFrequency, setShouldShowWeekFrequency] = useState(account.personalization.showWeekFrequency);
```

Comment:
Peut être undefined :

```suggestion
  const [shouldShowWeekFrequency, setShouldShowWeekFrequency] = useState(account.personalization.showWeekFrequency ?? false);
```

Reply:
Si l'option n'est pas définie, elle est déjà mise à `false` par défaut.